cairo_programs/cairo/hints/span_life.cairo

```diff
@@ -0,0 +1,4 @@
+fn main() {
```

Comment:
I don't understand the filename `span_life.cairo`
Maybe rename it to `span_slice.cairo` ?